src/main/java/org/ideoholic/curium/model/sendsms/dto/SendStaffSMSDto.java

```diff
@@ -0,0 +1,15 @@
+package org.ideoholic.curium.model.sendsms.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SendStaffSMSDto {
```

Comment:
Better use `SendNumberSMSDto` here.
Rename `SendNumberSMSDto` to `SendSMSDto` and add the required fields in the same DTO to be used in the current method.